content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,220 @@ We have deprecated the following cert-manager APIs:
 - `acme.cert-manager.io/v1alpha3`
 - `acme.cert-manager.io/v1beta1`
 
-These APIs will be removed in cert-manager `v1.6.0`.
-If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6.0`.
-See cert-manager [docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) for more detailed upgrade instructions.
+These APIs will be removed in cert-manager 1.6.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you have a cert-manager installation that is using or has previously used
+these deprecated APIs, you may need to upgrade your cert-manager custom
+resources and CRDs.
+
+This needs to be done before upgrading to cert-manager 1.6. See cert-manager
+[docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources)
+for more detailed upgrade instructions.
+
+{{% /pageinfo %}}
+
+## Helm chart: `securityContext` defaults to non-root
+
+The Helm chart [now](https://github.com/jetstack/cert-manager/pull/4036) follows
+the current Pod hardening best practices as defined by the Kyverno [pod-security
+restricted](https://kyverno.io/policies/pod-security/#restricted) policy.
```

Comment:
I can't stand links like `now` and `docs` and `here and `this`. 
I suggest not linking to the PR.
```suggestion
The Helm chart now follows
the [Kyverno restricted Pod security policy](https://kyverno.io/policies/pod-security/#restricted).
```